doc/reference/reference_lua/box_error.rst

```diff
@@ -77,10 +77,22 @@ Below is a list of all ``box.error`` functions.
 
     :except: whatever is specified in errcode-number.
 
+    ``box.error()`` is able to accept two types of arguments: either a pair of
```

Comment:
for easier skimming let's make this a bullet list:

```
    ``box.error()`` accepts two sets of arguments:
    
    * error code and reason (``box.error{code = 555, reason = 'Arbitrary message'}``), or
    * error object (``box.error(err)``).

    In both cases the error is promoted as the last error.
```

Reply:
fixed in d7a0ca6